blockchain/vm/internaltx_tracer.go

```diff
@@ -153,7 +153,8 @@ type InternalTxTrace struct {
 	Time  time.Duration      `json:"time,omitempty"`
 	Calls []*InternalTxTrace `json:"calls,omitempty"`
 
-	Reverted *RevertedInfo `json:"reverted,omitempty"`
+	RevertReason string        `json:"revertReason,omitempty"`
+	Reverted     *RevertedInfo `json:"reverted,omitempty"`
```

Comment:
Not sure, but probably `RevertedInfo` contain message which is reverted reason I guess. Was this updated?

Reply:
Yes it is. You can check the output example in this PR https://github.com/kaiachain/kaia/pull/15.
If reverted with message, then both `revertReason` and `reverted.message` are populated.